src/translations/en.json

```diff
@@ -2865,12 +2865,14 @@
             "caption": "Add Application Credential",
             "create": "Create",
             "domain": "Integration",
+            "name": "Name for display",
```

Comment:
Just name would be fine I think?

Reply:
hmm thinking now, it is going to confuse users I think? I wonder if we should only show name during updating, not creation.

Reply:
Renamed to Name.